test/unit/test-layout.cxx

```diff
@@ -105,12 +102,12 @@ TEST(LayoutTest, View)
 {
   int* data = new int[10];
 
-  using layout = RAJA::OffsetLayout<int, RAJA::PERM_I, int>;
+  using layout = RAJA::OffsetLayout<>;
 
   /*
    * View is constructed by passing in the layout.
    */
-  RAJA::View<int, layout> view(data, layout({1}, {10}));
+  RAJA::View<int, layout> view(data, layout({1}, std::array<int, 1>{10}));
```

Comment:
That's something I'm less than entirely happy about.  It's ambiguous whether the (array, array) or (array, layout) constructor should be called if the array isn't explicit.  It would be good to find a way around it, but I'm not sure what that way is.  It may be adding a factory function somewhere instead of a constructor I suppose.